pkg/s3wrapper/util.go

```diff
@@ -145,5 +156,63 @@ func CreateAndUploadMinimalIso(ctx context.Context, log logrus.FieldLogger,
 
 	// upload the minimal iso
 	log.Infof("Uploading minimal ISO (%s)", minimalIsoPath)
-	return api.UploadFileToPublicBucket(ctx, minimalIsoPath, minimalIsoObject)
+	if err := api.UploadFileToPublicBucket(ctx, minimalIsoPath, minimalIsoObject); err != nil {
+		return err
+	}
+
+	// Update version file in bucket
+	return updateISOTemplatesVersion(ctx, log, api)
+}
+
+// DoesHaveLatestMinimalTemplateVersion Returns true if latest version already exists in bucket; otherwise, false.
+func DoesHaveLatestMinimalTemplateVersion(ctx context.Context, log logrus.FieldLogger, api API) bool {
```

Comment:
Can you add a test for this?

Also this function name is enormous :laughing:
maybe `MinimalTemplateUpToDate` or just `HaveLatestMinimalTemplate`